common/browsered/field-validation-checks.js

```diff
@@ -47,37 +48,46 @@ module.exports.isPhoneNumber = function (value) {
   if (trimmedTelephoneNumber.length < 11 || !NUMBERS_ONLY.test(trimmedTelephoneNumber)) {
     return validationErrors.phoneNumber
   } else {
-    return false
+    return null
   }
 }
 
 module.exports.isHttps = function (value) {
   if (value.substr(0, 8) !== 'https://') {
     return validationErrors.https
   } else {
-    return false
+    return null
   }
 }
 
 module.exports.isBelowMaxAmount = value => {
   if (!module.exports.isCurrency(value) && parseFloat(value) >= MAX_AMOUNT) {
     return validationErrors.belowMaxAmount
   }
-  return false
+  return null
 }
 
 module.exports.isSortCode = value => {
   if (!/^\s?(\d{2}\s?-?\s?){2}\d{2}\s?$/.test(value)) {
     return validationErrors.sortCode
   } else {
-    return false
+    return null
   }
 }
 
 module.exports.isAccountNumber = value => {
-  if (!/^\s?(\d{2}\s?-?\s?){2,3}\d{2}\s?$/.test(value)) {
+  const trimmedAccountNumber = value.replace(/\s/g, '')
+  if ((trimmedAccountNumber.length < 6 || trimmedAccountNumber.length > 8) || !NUMBERS_ONLY.test(trimmedAccountNumber)) {
     return validationErrors.accountNumber
   } else {
-    return false
+    return null
+  }
+}
+
+module.exports.isRequiresAuthorisationChecked = value => {
+  if (value === false) {
```

Comment:
What happened to using .checked? This won't work for a wide variety of radio/check box values. 

Also isRequiresAuthorisationChecked is confusing and long why not isChecked?